docs/python_support/quickstart.md

```diff
@@ -0,0 +1,233 @@
+# Python support in OVMS - quickstart {#ovms_docs_python_support_quickstart}
+
+OpenVINO Model Server allows users to write custom processing nodes in Python, so they may have full control over what happens with the data reaching such node and what comes out of it.
+
+In this quickstart you will create a servable with a single custom Python code that will expect a string and return the same string, but all in uppercase. 
+
+To achieve that let's follow the steps:
+1. Prepare Docker Image And Workspace
+2. Write Python Code For The Server 
+3. Prepare Graph Configuration File
+4. Prepare Server Configuration File
+5. Deploy OpenVINO Model Server
+6. Create Client Application
+7. Send Requests From The Client
+
+### Step 1: Prepare Docker Image And Workspace
+
+Let's have all the work done in a new `workspace` directory. Also create the following subdirectories:
+- `workspace/models` (catalog that will be mounted to the deployment)
+- `workspace/models/python_model` (catalog with Python servable specification)
+
+You can do that in one go:
+
+```bash
+mkdir -p workspace/models/python_model && cd workspace
+```
+
+Since changing all the letters in the string to uppercase is a very basic example, the basic Python-enabled model server image without extra Python packages is sufficient. If you need some external modules, you need to add them to the image manually. 
+Until the image is published in next release, it needs to be built from sources:
+
+```bash
+git clone https://github.com/openvinotoolkit/model_server.git
+cd model_server
+make docker_build MEDIAPIPE_DISABLE=0 PYTHON_DISABLE=0 OV_USE_BINARY=1 RUN_TESTS=0
+cd ..
+```
+
+You will also need a client module so in your environment install a required dependency:
+```bash
+pip3 install tritonclient[grpc]
+```
+
+### Step 2: Write Python Code For The Server 
+
+Let's start with the server side code. Your job is to implement an `OvmsPythonModel` class. Model Server expects it to have at least `initialize` and `execute` methods.
+
+Since there's nothing to initialize, let's focus on `execute` method that will be called everytime model receives a request. The server will read inputs from that request and pass them to `execute` function as an `inputs` argument. 
```

Comment:
```suggestion
Since there's nothing to initialize, let's focus on `execute` method that will be called every time model receives a request. The server will read inputs from that request and pass them to `execute` function as an `inputs` argument. 
```